app/controllers/memories_controller.rb

```diff
@@ -36,6 +36,10 @@ def new
   def create
     @memory = current_user.memories.build(memory_params)
 
+    @memory.keyword_associations.split(',').each do |keyword|
```

Comment:
Unused block argument - `keyword`. You can omit the argument if you don't care about it.